{"name": "test002", "status": "failed", "statusDetails": {"message": "AssertionError: assert -1 == 5", "trace": "self = <testDemo.test002.TestClass object at 0x7fdfa898a2b0>\n\n    @allure.feature('allure测试')\n    @allure.story('test002')\n    def test002(self):\n        c = Calc()\n        a = c.jian(2, 3)\n>       assert a == 5\nE       assert -1 == 5\n\ntest002.py:27: AssertionError"}, "attachments": [{"name": "stdout", "source": "a398802c-468e-4db4-a7e6-770455af916c-attachment.txt", "type": "text/plain"}], "start": 1639302572132, "stop": 1639302572132, "uuid": "94f71cd9-fa65-4484-9821-e3862a44178c", "historyId": "1dbe753821056c3fd3e61cb346c54aad", "testCaseId": "750facfcfedbb613ccc33da634d5afa5", "fullName": "test002.TestClass#test002", "labels": [{"name": "story", "value": "test002"}, {"name": "feature", "value": "allure测试"}, {"name": "suite", "value": "test002"}, {"name": "subSuite", "value": "TestClass"}, {"name": "host", "value": "MBP.local"}, {"name": "thread", "value": "38843-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test002"}]}